content/mactynow/drone-s3cache/index.md

```diff
@@ -0,0 +1,110 @@
+---
+date: 2017-06-29T00:00:00+00:00
+title: Drone-s3cache
+author: mactynow
+tags: [ s3, cache ]
+repo: mactynow/drone-s3cache
+logo: amazon_s3.svg
+image: quay.io/honestbee/drone-kubernetes
+---
+
+{{% alert error %}}
+This plugin requires an AWS account. 
+{{% /alert %}}
+
+This plugin can be used to preserve files and directories between builds and store them on AWS s3. The below pipeline configuration demonstrates simple usage.
+
+{{% alert info %}}
+This plugin is inspired by and contains bits of https://github.com/Drillster/drone-volume-cache. If you don't care about storing your cache somewhere else than on your host, then use their work!
+{{% /alert %}}
+
+Example pipeline configuration:
+
+```yaml
+pipeline:
+  restore-cache:
+    image: mactynow/drone-s3cache
+    aws_access_key_id: ${AWS_ACCESS_KEY_ID}
```

Comment:
Can you pass secrets like this @tboerger, or do you have to use `secrets: [AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY]`?

Reply:
As commented above within my review this won't work anymore.